packages/Sandblocks-Core/SBTabView.class.st

```diff
@@ -371,7 +377,7 @@ SBTabView >> removeCurrentTab [
 { #category : #tabs }
 SBTabView >> setActive: aNamedBlock [
 
-	self sandblockEditor do: 
+	SBEditor current do: 
```

Comment:
why was this change necessary?

Reply:
(also in the other places)

Reply:
when creating the exploriants block, the block is first created and then added to the editor. should it be the other way around?